environments/staging/readme.md

```diff
@@ -20,6 +20,43 @@
 
 -----------
 
+- Version: Aragon Fundraising 1.0.0-rc.1
+- Start date: December 12th 2019, 11.00am CEST
+- Finish date: December 12th 2019, 7.30pm CEST
+- Deployer: ([@sohkai](https://github.com/sohkai)) [`0x15bdCb6651f5a5e9Cef07b2a981dd270BA505051`](https://rinkeby.etherscan.io/address/0x15bdCb6651f5a5e9Cef07b2a981dd270BA505051)
+
+### Aragon Fundraising
+
+Repo: https://github.com/AragonBlack/fundraising
+
+Command (`aragon-fundraising`):
+```
+aragon apm publish major --files dist --environment rinkeby
+```
+
+Command (`bancor-formula`, `batched-bancor-market-maker`, `presale`, `tap`):
+```
+aragon apm publish major --environment rinkeby
```

Comment:
Or `npm run apm:publish:rinkeby:major`

Reply:
Clarified this more; just said it duplicated the rinkeby deployment :).